azure-pipelines-microbuild.yml

```diff
@@ -1,77 +1,75 @@
-resources:
-- repo: self
-  clean: true
-queue:
-  name: VSEng-MicroBuildVS2019
-  demands: Cmd
-variables:
-  BuildConfiguration: Release
-  TeamName: Roslyn
-  _DotNetArtifactsCategory: .NETCore
+# Branches that trigger a build on commit
+trigger:
+  - master
+  - 2.9.x
 
-steps:
-- task: ms-vseng.MicroBuildTasks.30666190-6959-11e5-9f96-f56098202fef.MicroBuildSigningPlugin@1
-  displayName: Install Signing Plugin
-  inputs:
-    signType: $(SignType)
-    esrpSigning: true
-  condition: and(succeeded(), ne(variables['SignType'], ''))
+variables:
+  - name: _TeamName
+    value: Roslyn
+  - name: _DotNetArtifactsCategory
+    value: .NETCore
 
-- script: eng\common\CIBuild.cmd 
-            -configuration $(BuildConfiguration)
-            /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
-            /p:DotNetSignType=$(SignType)
-            /p:DotNetSymbolServerTokenMsdl=$(microsoft-symbol-server-pat)
-            /p:DotNetSymbolServerTokenSymWeb=$(symweb-symbol-server-pat)
-            /p:PublishToSymbolServer=true
-            /p:DotNetPublishToBlobFeed=true
-            /p:DotNetPublishBlobFeedKey=$(dotnetfeed-storage-access-key-1)
-            /p:DotNetPublishBlobFeedUrl=https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
-            /p:DotNetArtifactsCategory=$(_DotNetArtifactsCategory)
-            /p:DotnetPublishUsingPipelines=true
+stages:
+- stage: build
   displayName: Build
+  jobs:
+  - template: /eng/common/templates/jobs/jobs.yml
+    parameters:
+      enableMicrobuild: true
+      enablePublishBuildArtifacts: true
+      enablePublishTestResults: true
+      enablePublishBuildAssets: true
+      enablePublishUsingPipelines: true
+      enableTelemetry: true
+      jobs:
+      - job: Signed_Build
+        pool:
+          name: NetCoreInternal-Pool
+          queue: BuildPool.Windows.10.Amd64.VS2019
+        variables:
+          - group: DotNet-Blob-Feed
+          - group: Publish-Build-Assets
+          - name: _BuildConfig
+            value: Release
+          - name: _SignType
+            value: real
+        steps:
+        - checkout: self
+          clean: true
+        - script: eng\common\CIBuild.cmd 
+                    -configuration $(_BuildConfig)
+                    -binaryLog
```

Comment:
>-binaryLog [](start = 20, length = 10)

`-binaryLog` shouldn't be necessary. It's the default for CI.

Reply:
ah yes